.github/workflows/static-analysis.properties.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Static analysis",
+  "description": "Perform static analysis on the source code and tests",
+  "iconName": "doctrine-logo",
+  "categories": [
+    "PHP"
+  ],
+  "filePatterns": [
+    "^phpstan\\.neon(?:\\.dist)$",
+    "^psalm\\.xml$"
+  ]
+}
```

Comment:
That file should not be copied here, it's there just so you can see a nice icon while using the GUI to setup the action